modules/efk-logging-fluentd-undefined.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-fluentd.adoc
+
+[id="efk-logging-fluentd-undefined_{context}"]
+= Configuring how the log collector normalizes logs 
+
+Cluster Logging uses the link:https://github.com/ViaQ/fluent-plugin-viaq_data_model[ViaQ common data model] to deal with problem that arise from conflicts 
+and inconsistencies in log data collected from different subsystems.   
+
+If you use the `MERGE_JSON_LOG` feature (MERGE_JSON_LOG=true), it can be extremely useful to have your applications log their output in JSON, and have cluster logging automatically parse and index the data in Kibana. However, this leads to several problems, including:
```

Comment:
Thought you might want to say "and have log collectors automatically parse and index the data in Elasticsearch" here.

Reply:
@QiaolingTang Thank you. I made the change.